spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/integration/IntegrationGraphEndpointAutoConfiguration.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2012-2018 the original author or authors.
```

Comment:
Is it a rule in Spring Boot to include `2012` as well to new classes?

Reply:
yes